api/views.py

```diff
@@ -65,7 +66,42 @@ class RegistrationAPIViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
     serializer_class = RegistrationSerializer
 
 
+class VolunteerFilter(filters.FilterSet):
+    time_volunteered = filters.NumberFilter(lookup_expr='exact')
+    time_volunteered__gt = filters.NumberFilter(method='time_volunteered_gt')
```

Comment:
Just wondering, doesn't this variable and the method with the same name collide (because of the naming thing)? Asking just in case it's not intended (if it works then it's ok)

Reply:
They don't, look at the number of underlines